server/router.js

```diff
@@ -125,7 +130,9 @@ function sendSPA (route, req, res, params, state) {
       return res.end(require('./page-debug')(contents, serializeJS(frozenState)))
     })
   }
-  return res.end(page(contents, serializeJS(frozenState)))
+  if (!metadata) metadata = {}
+  metadata.route = route
+  return res.end(page(contents, metadata, serializeJS(frozenState)))
```

Comment:
Ah yeah, I thought about this for a minute. Because page.js is global (for every route) it felt better to treat the metadata property as a metadata of the page, rather than of the specific archive route. So if we have user profiles, for example, the API stays the same